packages/react-data-grid-addons/src/editors/DropDownEditor.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { Column } from 'react-data-grid';
+
+interface Option {
+  id: string;
+  title?: string;
+  value?: string;
+  text?: string;
+}
+
+interface Props {
+  value?: string | string[] | number;
+  column: Column;
+  options: (string | Option)[];
+  onBlur: React.FocusEventHandler<HTMLSelectElement>;
+}
+
+interface State {
+  value?: string | string[] | number;
+}
```

Comment:
This might be better written like this:

```suggestion
interface State {
  value?: string | string[] | number;
}
```